src/services/types.ts

```diff
@@ -71,6 +71,10 @@ namespace ts {
         getLineAndCharacterOfPosition(pos: number): LineAndCharacter;
     }
 
+    export interface SourceMapSource {
+        /* @internal */ getLineAndCharacterOfPosition(pos: number): LineAndCharacter;

```

Comment:
Removing `/* @internal */` since removing it then it will essentially force calling `createSoruceMapSource()` which is, as you point out, a good thing.